rotkehlchen/db/updates.py

```diff
@@ -88,49 +90,111 @@ def update_spam_assets(self, local_version: int, to_version: int) -> None:
         locally
         """
         for version in range(local_version + 1, to_version + 1):
-            file_url = SPAM_ASSETS_URL.format(branch=self.branch, version=version)
-            try:
-                updates = query_file(file_url, is_json=True)
-            except RemoteError as e:
-                log.warning(f'Failed to update spam assets due to {str(e)}')
-                break
-            updated_assets = updates.get('assets')
-            if updated_assets is None:
-                log.error(f'Remote update {file_url} does not contain the assets key')
+            ok, updated_assets = self._get_file_content(
```

Comment:
nitpick. Too short a var name. 

```suggestion
            success, updated_assets = self._get_file_content(
```